src/Compilers/VisualBasic/Portable/Emit/EditAndContinue/VisualBasicSymbolMatcher.vb

```diff
@@ -692,6 +692,11 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Emit
             End Function
 
             Public Overrides Function VisitNamedType(type As NamedTypeSymbol) As Symbol
+                If (type.IsTupleType) Then
```

Comment:
I have reviewed scenarios mentioned in the discussion from the past code review [https://github.com/dotnet/roslyn/pull/19114](https://github.com/dotnet/roslyn/pull/19114)
There are EditAndContinueTests.cs with tuples in the foreach statement. We attempted to change their names or types.

1. We still do not have a match for them, i.e. extra locals appear.
2. We still have a rude edit warning for this situation. This prevents us from run-time errors.
3. We already covered the situation with [https://github.com/dotnet/roslyn/issues/19154](https://github.com/dotnet/roslyn/issues/19154)

---
In reply to: [116639124](https://github.com/dotnet/roslyn/pull/19283#discussion_r116639124) [](ancestors = 116639124,116339869)